src/components/ResponsiveGrid.tsx

```diff
@@ -10,72 +10,47 @@
  * governing permissions and limitations under the License.
  */
 import { MapTo, withComponentMappingContext } from '../core/ComponentMapping';
-import {
-  AllowedComponentsContainer,
-  AllowedComponentsProperties,
-} from './allowedcomponents/AllowedComponentsContainer';
-import { ContainerState } from './Container';
-import { PlaceHolderModel } from './ContainerPlaceholder';
+import { AllowedComponentsContainer } from './AllowedComponentsContainer';
 import { EditConfig } from '../core/EditableComponent';
-import { Constants } from '../Constants';
-
-export interface ResponsiveGridProperties extends AllowedComponentsProperties {
-  gridClassNames: string;
-  columnClassNames: { [key: string]: string };
-}
-
-export class ResponsiveGrid<
-  P extends ResponsiveGridProperties,
-  S extends ContainerState,
-> extends AllowedComponentsContainer<P, S> {
-  public static defaultProps = {
-    _allowedComponentPlaceholderListEmptyLabel: 'No allowed components',
-    cqItems: {},
-    cqItemsOrder: [],
-    cqPath: '',
-    title: 'Layout Container',
+import {ModelProps, ResponsiveGridProps} from "../types/AEMModel";
+import React from 'react';
+import {ClassNames} from "../constants/classnames.constants";
+
+type Props = {
+  title?: string;
+} & ResponsiveGridProps;
+
+export const ResponsiveGrid = ({
+  title = 'Layout Container',
+  ...props
+}: Props) => {
+  const getItemCustomProps = function (itemKey: string, itemProps: ModelProps) {
```

Comment:
If this would only ever have classNames, should we consider something more specific to it like `getCustomClasses` and return the string directly?